python/cudf/cudf/_lib/nvtext/tokenize.pyx

```diff
@@ -122,3 +126,37 @@ def detokenize(Column strings, Column indices, object py_separator):
         )
 
     return Column.from_unique_ptr(move(c_result))
+
+
+cdef class Tokenize_Vocabulary:
```

Comment:
No underscores needed in Python class names. Just a capital letter for each word.
```suggestion
cdef class TokenizeVocabulary:
```